tensorflow_recommenders_addons/dynamic_embedding/core/kernels/redis_impl/json.cc

```diff
@@ -402,8 +402,8 @@ json_value *json_parse_ex(json_settings *settings, const json_char *json,
             case json_object:
 
               if (state.first_pass)
-                (*(json_char *
-                   __attribute((__may_alias__)) *)&top->u.object.values) +=
+                (*(json_char *__attribute((
```

Comment:
I believe you should use a mismatch format method or tools.

Reply:
I have used clang-format 13.0, I reformatted the code using clang-format 9.